ploneorg/theme/theme/plone-open-garden.html

```diff
@@ -2195,30 +2196,6 @@ <h2>We love PLOG.</h2>
     </div>
   </div>
 
-  <div class="row-fluid vertical-spacer">
-    <div class="span6 offset3 text-center vertical-spacer">
-      <h2>Included training</h2>
-      <p>
-          This year we are offering <strong>awesome free training</strong> by the developers making Plone's future come to life:
-        <ul>
-            <li>
-                Python 3 porting training, by Philip Bauer; 10 am - 1 pm, April 13, 14, or 15
-            </li>
-            <li>
-                React/Plone-React training, by Rob Gietema; 10 am - 1pm, April 16, 17, or 18
-            </li>
-        </ul>
-        Just sign up for these classes on our registration form!
-        </p>
-        <div class="row-fluid">
-            <div class="span8 offset2">
-                <a class="btn btn-flat btn-primary" href="/plog-registration-form">
-                    Register now
-                </a>
-            </div>
-        </div>
-    </div>
-  </div>
```

Comment:
Are you removing this section permanently? (ie are we not offering free training?)

Reply:
Free training is included, I think the idea is to make that a permanent feature. I added a mention of it to the "What is PLOG?" section at the top, and to the "Speaker's Corner" section which I renamed "Training & Discussions". 

I did intend to remove this section from this location permanently. It interrupts the flow between "We love PLOG"..."Here are some Plonistas who have fallen in love with PLOG", and the section with people and quotes. Also I want to avoid giving the specifics of training classes, which a) would have to be edited every year and b) haven't been completely decided yet anyway. (We've got the basic ideas decided, but have not nailed down schedule or trainers, which will take a while.) And we are not going to ask people to sign up for trainings on the registration form. It is expected that almost everyone will take them, with a few sitting to the side doing their own projects.

That said, I agree that it would be good to really highlight the fact that training is included. What about adding a centered section "Training Included!" in between "Being part of PLOG is easy." and "We love PLOG."? I'll work on that.

Reply:
ok

Reply:
I have created a "Training Included!" section. Fingers crossed I didn't make any HTML mistakes.